ha/note.c

```diff
@@ -180,6 +182,24 @@ static void ha_state_accept(struct m0_confc         *confc,
 			if (!ignore_same_state ||
 			    prev_ha_state != obj->co_ha_state)
 				m0_chan_broadcast(&obj->co_ha_chan);
+		} else if ((obj == NULL) &&
+		           (m0_fid_tget(&note->nv_note[i].no_id) == 'r')) {
+				M0_LOG(M0_DEBUG, FID_F" is process FID and not found in conc",
+					FID_P(&note->nv_note[i].no_id));
+				M0_LOG(M0_DEBUG,"Add the entry to conf cache");
+				rc = m0_conf_obj_find(cache, &note->nv_note[i].no_id, 
+						&obj);
+				if (rc == 0) {
+					obj->co_status = M0_CS_READY;
+					prev_ha_state = obj->co_ha_state;
```

Comment:
New function can be created from line 194 to 199 and  same can get reused by code from line 180 to 184